barretenberg/cpp/src/barretenberg/transcript/transcript.hpp

```diff
@@ -128,7 +146,7 @@ template <typename TranscriptParams> class BaseTranscript {
      * to the current challenge buffer to set up next function call.
      * @return std::array<Fr, HASH_OUTPUT_SIZE>
      */
-    [[nodiscard]] Fr get_next_challenge_buffer()
+    [[nodiscard]] std::array<Fr, 2> get_next_duplex_challenge_buffer(size_t num_challenges)
```

Comment:
please update this so that it checks num_challenges is 1 or 2, or at least update some comment

Reply:
done